pkgs/development/python-modules/liblarch/default.nix

```
@@ -0,0 +1,46 @@
+{ stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, python
+, pygobject3
+, xvfb_run
+, gobject-introspection
+, gtk3
+}:
+
+buildPythonPackage rec {
+  version = "3.0.1";
+  pname = "liblarch";
```

python2 fails to import the modules
```suggestion
  pname = "liblarch";
  disabled = isPy27;
```
```
ImportError: Failed to import test module: tests.test_filteredtree
Traceback (most recent call last):
  File "/nix/store/hngp562dxvyj3pfvph1nyqnr03sswxy5-python-2.7.18/lib/python2.7/unittest/loader.py", line 254, in _find_tes>
    module = self._get_module_from_name(name)
  File "/nix/store/hngp562dxvyj3pfvph1nyqnr03sswxy5-python-2.7.18/lib/python2.7/unittest/loader.py", line 232, in _get_modu>
    __import__(name)
  File "/build/source/tests/test_filteredtree.py", line 21, in <module>
    from liblarch.treenode import _Node
  File "/build/source/liblarch/__init__.py", line 21, in <module>
    from .tree import MainTree
  File "/build/source/liblarch/tree.py", line 416
    print(output, end=' ')
                     ^
SyntaxError: invalid syntax
```

---

changed but found `disabled = pythonOlder "3.5.0";` nicer.

---

effectively the same thing :)